src/cordova/android/container/setup.sh

```diff
@@ -26,13 +26,13 @@ set -eu
 # Download Android Command Line Tools:
```

Comment:
If this lives in `android/container`, where would the macOS setup script live?

I think we should move this out of the container, since it's to be used in other contexts.
Perhaps `android/build/setup_linux.sh`, or `android/setup_linux_build.sh`